packages/bot-web-ui/src/components/chart/chart.scss

```diff
@@ -26,4 +26,167 @@
+                    display: flex;
+                    border-radius: 50%;
+                    justify-content: center;
+                    align-items: center;
+                    margin: 0.8rem;
+                    opacity: 0.75;
+
+                    &__menu {
+                        &__timeperiod {
+                            top: 0.8rem;
+                            left: 0.8rem;
+                        }
+                        & > .ic-icon {
+                            top: 0.6rem;
+                        }
+                    }
+                }
+            }
+            .app-contents .ciq-menu {
+                margin: 0;
+            }
+        }
+    }
+
+    .smartcharts-mobile {
+        .cq-context {
+            z-index: 99;
+        }
+
+        & .cq-chart-title .cq-menu-dropdown {
+            position: fixed;
+            height: 100% !important;
+            .sc-dialog {
+                height: 100% !important;
+                &__body {
+                    height: inherit !important;
+                    .sc-mcd {
+                        height: inherit !important;
+                        min-width: auto !important;
+                    }
+                }
+            }
+        }
+    }
+
+    /** @define mobile-wrapper; weak */
+    .mobile-wrapper {
+        padding: 0 0.8rem;
+        display: flex;
+        flex-direction: column;
+
+        &__content-loader {
+            position: absolute;
+            height: 100%;
+            width: 100%;
+            left: 0;
+            bottom: -0.8rem;
+
+            svg {
+                height: 100%;
+                width: 100%;
+            }
+        }
+
+        /** @define dc-collapsible */
+        @include mobile {
+            .dc-collapsible {
+                position: absolute;
+                width: 100%;
+                margin: 0 auto;
+                left: 0;
+                bottom: 0;
+                z-index: 999;
+
+                .toolbar__section {
+                    justify-content: center;
+
+                    .animation__container {
+                        width: 17.6rem;
+                    }
+                }
+
+                &:before {
+                    content: '';
+                    position: absolute;
+                    pointer-events: none;
+                    opacity: 1;
+                    z-index: -1;
+                    border-radius: $BORDER_RADIUS;
+                    top: 0;
+                    left: 0;
+                    width: 100%;
+                    height: 100%;
+                    transition: opacity 0.25s;
+                    background: var(--general-section-1);
+                }
+                &--is-expanded {
+                    background: transparent;
+
+                    &:before {
+                        opacity: 0.75;
+                    }
+                }
+            }
+        }
+    }
+}
+
+.cq-modal-dropdown.stxMenuActive {
+    left: 0;
+    top: 0;
```

Comment:
is this class required i cannot see this in the element or can we nest this class ?? @maryiafrantsava-binary

Reply:
@rupato-deriv to see this class you should 
1)go to Charts tab
2)open dropdown markets